src/developer-quickstart.md

```diff
@@ -0,0 +1,441 @@
+
+Before we begin, it may be helpful to understand terminology that will used throughout the docs and how they relate to each other:
+
+- **Fuel**: the Fuel blockchain.
+- **FuelVM**: the virtual machine powering Fuel.
+- **Sway**: the domain-specific language crafted for the FuelVM; it is inspired by Rust.
+- **Forc**: the build system and package manager for Sway, similar to Cargo for Rust.
+
+## Understand Sway Program Types
+
+There are four types of Sway programs:
+
+- `contract`
+- `predicate`
+- `script`
+- `library`
+
+Contracts, predicates, and scripts can produce artifacts usable on the blockchain, while a library is simply a project designed for code reuse and is not directly deployable.
+
+|             | deployable on the blockchain:       | can have state:      | callable on the blockchain: | designed for code reuse: |
+|-------------|-------------------------------------|----------------------|-----------------------------|--------------------------|
+| contract  | ✅                                  | ✅                   | ✅                          | ❌                        |
+| predicate | ✅                                  | ❌                    | ❌                           | ❌                        |
+| script    | ❌                                   | ❌                    | ❌                           | ❌                        |
+| library   | ✅ (via a contract or predicate) | ✅ (via a contract) | ❌                           | ✅                       |
+
+See [the chapter on program types](../sway-program-types/index.md) for more information.
+
+## Your First Sway Project
+
+We'll build a simple counter contract with two functions: one to increment the counter, and one to return the value of the counter.
+
+A few pieces of info that will be helpful before moving on:
+
+- The main features of a smart contract that differentiate it from scripts or predicates are that it is callable and stateful.
+- A script is runnable bytecode on the chain which can call contracts to perform some task. It does not represent ownership of any resources and it cannot be called by a contract.
+
+### Writing the Contract
+
+First, let's [install the Sway toolchain](https://github.com/FuelLabs/fuelup). Then with `forc` installed, create a contract project:
+
+```sh
+forc new counter_dapp
+```
+
+Here is the project that `Forc` has initialized:
+
+```console
+$ cd counter_dapp
+$ tree .
+├── Cargo.toml
+├── Forc.toml
+├── src
+│   └── main.sw
+└── tests
+    └── harness.rs
+```
+
+`Forc.toml` is the _manifest file_ (similar to `Cargo.toml` for Cargo or `package.json` for Node) and defines project metadata such as the project name and dependencies. You'll notice a `cargo.toml` file because `forc` uses `cargo` under the hood.
+
+We'll be writing our code in the `src/main.sw`.
+
+Open your project in a code editor and delete the boilerplate code in `src/main.sw`.
```

Comment:
It says to delete the `src/main.sw` but don’t say to recreate it

Reply:
I just say to delete the code in the file, not the file itself